src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc

```diff
@@ -154,13 +154,12 @@ class PickFirst : public LoadBalancingPolicy {
 
   /// Lock and data used to capture snapshots of this channels child
   /// channels and subchannels. This data is consumed by channelz.
-  gpr_mu child_refs_mu_;
+  grpc_core::Mutex child_refs_mu_;
```

Comment:
This is already in the `grpc_core` namespace, so the prefix shouldn't be necessary.

Reply:
Sure, done.